clinica/iotools/converters/adni_to_bids/adni_modalities/adni_av45_fbb_pet.py

```diff
@@ -33,6 +34,11 @@ def convert_adni_av45_fbb_pet(
     mod_to_update : bool
         If True, pre-existing images in the BIDS directory
         will be erased and extracted again.
+
+    n_procs : int, optional
+        The requested number of processes.
+        If specified, it should be between 1 and the number of available CPUs.
+        Default=1.
```

Comment:
Same for the docstring

```suggestion
    n_procs : int, default=1
        The requested number of processes.
        If specified, it should be between 1 and the number of available CPUs.
        Default=1.
```